extra-files/cloud_setup/shed_tool_conf.xml.cloud

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
```

Comment:
This is a left-over from the time before labels could be created automatically; this file is not even being copied over currently.